gui/gui.py

```diff
@@ -33,6 +33,7 @@
 matplotlib.use('TkAgg')
 
 def get_control_mode(window):
+    # TODO: Is this still how we are retrieving control_mode?
```

Comment:
Nope you're right. We can prob rename this function to get_sim_control_mode since this is only used in the simulated no threads version OR we can add a flag isSim which will chose whether it does the current code vs read robot_data. This would be done on a diff branch tho